packages/compiler/src/core/types.ts

```diff
@@ -1672,6 +1673,14 @@ export interface LibraryLocationContext {
   metadata: ModuleLibraryMetadata;
 }
 
+// TODO better name?
+export interface Library {
```

Comment:
LibraryDefinition?

Reply:
there is already `LibraryDefinition` for what is the result of `createLibrary`

Reply:
`LibrarySpecification`?